### ISSUE TITLE
".recaf" Configuration directory

### ISSUE THREAD

**Issue by pedroflor** [Col-E/Recaf#104]
Hi Col-E, I'm not trying to be an spammer... I just love your software and I'm trying to give some suggestions.

## ".recaf" Configuration directory

all the config files (.json) and others, should have their own configuration directory; for example: ".recaf" (dot recaf)

Currently, the "current" directory of "recaf-x.jar" creates a bunch of .json configuration files, generating some "visual noise".

Thanks!

**Comment by andylizi** [comment]
> Currently, the "current" directory of "recaf-x.jar" creates a bunch of .json configuration files, generating some "visual noise".

Agreed. But instead of moving all of them to a new directory, why not just combine them to a single JSON file? It can archive the same "visual" result with a much cleaner design.

**Comment by Col-E** [comment]
> Hi Col-E, I'm not trying to be an spammer

Nonsense, I appreciate all the issues and ideas people provide. I'm glad you like the tool enough to contribute ideas.

As for the issue of making a folder vs combining the files I will probably go with a folder. Here's the pros that I see for each:

**pro**: config folder

* Hides config clutter in some directory 
* Minimal code changes to implement 
* Options are easier to find due to categorization vs a single file
* Easy to migrate existing config to new directory location

**pro**: single config file

* Hides config clutter in single file 

So in my mind the real question is where should the folder go? The home directory? Adjacent to the recaf jar?

**Comment by pedroflor** [comment]
Thanks!
